library/MedEx/MedEx_background.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
```

Comment:
Do note for this script and the script above that anybody will be able to run this from a web browser. ie localhost://library/MedEx/MedEx_background.php .

Not the end of the world, but just important to think about (for example, if bringing in parameters, which you are not, need to ensure no vulnerabilities are opened). Also when echo stuff here, would be as generic as possible so bad actors can't interrogate the system (and put the details in the log).

Note a safer option would be to encapsulate these in functions and using the openemr background service engine.